net/frr/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		frr
+PLUGIN_VERSION=		1.0.0
```

Comment:
since we use PLUGIN_DEVEL, I'd rather have a version < 1

Reply:
ok